dm/tests/import_v10x/run.sh

```diff
@@ -88,7 +88,7 @@ function run() {
 	run_sql "show create table \`dm_meta\`.\`test_syncer_checkpoint\`" $TIDB_PORT $TIDB_PASSWORD
 	check_contains "\`exit_safe_binlog_name\` varchar(128) DEFAULT ''"
 	# different TiDB will output DEFAULT 0 or DEFAULT '0'
-	check_contains "\`exit_safe_binlog_pos\` int(10) unsigned DEFAULT "
+	check_contains "\`exit_safe_binlog_pos\` int unsigned DEFAULT "
```

Comment:
```suggestion
	check_contains "\`exit_safe_binlog_pos\` int"
```

Also handle different versions of downstream TiDB